docs/best-practices.md

```diff
@@ -0,0 +1,28 @@
+Best Practices
+---------
+
+This page is a placeholder for best practices we recommend you to review while implementing Vue InstantSearch.
```

Comment:
Not sure, I will try to inspire from Vue.js official docs I think.
What we could add though is security best practices.